assets/scss/modules/_modal-dialog.scss

```diff
@@ -0,0 +1,144 @@
+/* 
+Modal Dialog
+
+Experimental:  An accessibility compliant & responsive modal dialog box module.
+
+**Before considering a [modal dialog box](https://designpatterns.hackpad.com/Modal-dialog-boxes-ZJNqssq4Dll), make sure that you have strong evidence that:**
+* There is a real user need to have attention focused on a single piece of content;
+* That need cannot be met by taking the user to a new page in the same flow that contains the crucial piece of content;
+
+
+Markup:
```

Comment:
This is a massive example and very specific to HMRC. Can it be made smaller and have more generic content?